sds_data_manager/utils/stackbuilder.py

```diff
@@ -145,21 +138,22 @@ def build_sds(
         #     ...
         # )
 
-        processing_stack.ProcessingStep(
-            scope,
-            f"{instrument}Processing",
-            env=env,
+        # NOTE: processing_stack.ProcessingStep was giving this
+        # error:
+        # AttributeError: 'CfnJobDefinition' object has no attribute
+        # 'attr_job_definition_arn'
+        # Replaced it with batch job stack
+        batch_job_stack.BatchJobStack(
```

Comment:
Can you just call `FargateBatchResources()` directly here instead of creating a new stack? This new stack is just passing things through immediately.

Reply:
I don't know if I can since that is a construct but let me try. That would simplify it.

Reply:
Turn that construct into a Stack then? Not sure it needs to be one versus the other.